libraries/MySensors/MyConfig.h

```diff
@@ -35,11 +35,6 @@
 // final sketch but is helpful to see what is actually is happening during development
 //#define MY_DEBUG
 
-// Enable MY_DEBUG_VERBOSE flag for verbose debug prints related to RF24 radio.
-// Requires DEBUG to be enabled.
-// This will add even more to the size of the final sketch!
-//#define MY_DEBUG_VERBOSE
-
```

Comment:
I suggest adding a commented section, like MY_DEBUG_VERBOSE_SIGNING below. Perhaps even add doxygen tags to these so that the debug features gets documented. But that can be made in a separate commit. I encourage everyone to improve on the doxygen parts :)